### PR TITLE
Use artifact from qpc_img_poll for latest image

### LINKED ISSUE #78: Internal registry not providing up-to date images
We are getting strange results from Jenkins nightly automation because the image getting pulled from the internal registry is not up to date!!!

We should just use the artifact from the job that builds the images to make sure we are testing the current state of master.

### PR DESCRIPTION
**Pull request by kdelee**
We were running into troubles with using the internal registry.

Closes #78

### CODE REVIEW

File: jjb/jobs/qpc-nightly-automation.yaml [hunk 1]
```diff
@@ -40,11 +46,11 @@
                 echo "INSECURE_REGISTRY=\"--insecure-registry ${DOCKER_REGISTRY}\"" >> docker.conf
                 sudo cp docker.conf /etc/sysconfig/docker
                 sudo systemctl start docker
-                sudo docker pull "${DOCKER_REGISTRY}/quipucords/quipucords:1.0.0"
+                sudo docker load -i ${WORKSPACE}/quipucords*.tar.gz
                 # make log dir to save server logs
                 mkdir ${WORKSPACE}/log
                 # run container binding local ssh keys and log dir to the container
-                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -v ${WORKSPACE}/log:/var/log -i "${DOCKER_REGISTRY}/quipucords/quipucords:latest"
+                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -v ${WORKSPACE}/log:/var/log -i "quipucords:1.0.0"
```
Comment (kdelee) [inline jjb/jobs/qpc-nightly-automation.yaml]: @chambridge `1.0.0` is the correct tag to use, right? Ot is there a different tag for whatever is the latest master version?

File: jjb/jobs/qpc-nightly-automation.yaml [hunk 2]
```diff
@@ -19,10 +19,16 @@
...
+            project: qpc_docker_img_poll
+            filter: quipucords.1.0.0.tar.gz
+            target: ${WORKSPACE}
+            stable: true
+
         - config-file-provider:
             files:
-              - file-id: '62cf0ccc-220e-4177-9eab-f39701bff8d7'
-                target: 'camayoc/config.yaml'
+                 - file-id: '62cf0ccc-220e-4177-9eab-f39701bff8d7'
```
Comment (chambridge) [inline jjb/jobs/qpc-nightly-automation.yaml]: Should these have been indented?

File: jjb/jobs/qpc-nightly-automation.yaml (hunk 1 above)
Comment (chambridge) [reply jjb/jobs/qpc-nightly-automation.yaml]: `1.0.0` is correct.

**Review by chambridge — APPROVED** [review]
Looks good. One question about the config file tabbing.

**Review by elyezer — APPROVED** [review]
LGTM

**Comment by kdelee** [comment]
Fixed the inconsistent spacing!
Thanks.

**Pull request merged**